src/Sylius/Component/Core/spec/Model/TaxonSpec.php

```diff
@@ -11,25 +11,55 @@
 
 namespace spec\Sylius\Component\Core\Model;
 
+use Doctrine\Common\Collections\Collection;
 use PhpSpec\ObjectBehavior;
-use Sylius\Component\Core\Model\ImageInterface;
+use Sylius\Component\Core\Model\Taxon;
+use Sylius\Component\Core\Model\TaxonImageInterface;
 use Sylius\Component\Taxonomy\Model\TaxonInterface;
 
+/**
+ * @mixin Taxon
+ *
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
 final class TaxonSpec extends ObjectBehavior
 {
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Component\Core\Model\Taxon');
+        $this->shouldHaveType(Taxon::class);
     }
 
-    function it_is_Sylius_Taxon()
+    function it_is_Sylius_taxon()
```

Comment:
Still think it should be ``it_is_a_taxon`` or ``it_implements_taxon_interface`` :(